servers/rendering/rendering_device.cpp

```diff
@@ -903,7 +903,7 @@ RID RenderingDevice::texture_create_shared(const TextureView &p_view, RID p_with
 
 	// Create view.
 
-	Texture texture = *src_texture;
+	Texture texture = src_texture->duplicate_as_shared_texture();
 	texture.shared_fallback = nullptr;
```

Comment:
I guess `duplicate_as_shared_resource` should also avoid copying `shared_fallback` so we don't have to manually set it to nullptr right after doing the copy